src/accounts/accountModel.java

```diff
@@ -0,0 +1,121 @@
+package accounts;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.ResourceBundle;
+import javafx.fxml.Initializable;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ *
+ * @author aswecker
+ */
+public class accountModel implements Initializable {
```

Comment:
Class names should start with an uppercase letter. Also, this class shouldn't implement Initializable; that's just for views and their associated controllers.